src/componenets/Dropdown.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        open: false,
+        items: this.props.items,
+        header: this.props.header
+    };
+    this.selectItem = this.selectItem.bind(this);
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+  }   
+  toggleDropdown() {
+    this.setState( (prevState) => ({
+      open: !prevState.open
+    }));
+  }
+
+  selectItem(item) {
+    this.setState({
+      header: item,
+      open: false
+    });
+  }
+
+  renderItems() {
+    if (!this.props.items) {
```

Comment:
Please build an empty state component and render it when there are no available items after a search has been done